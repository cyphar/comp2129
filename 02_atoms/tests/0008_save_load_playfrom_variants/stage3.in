LOAD save0.as
PLAYFROM 0
DISPLAY
STAT
