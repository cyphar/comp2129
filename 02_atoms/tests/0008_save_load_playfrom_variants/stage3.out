Game Loaded

Game Ready
Red's Turn


+-----------------------+
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
+-----------------------+

Player Red:
Grid Count: 0

Player Green:
Grid Count: 0

Player Purple:
Grid Count: 0

Player Blue:
Grid Count: 0

Player Yellow:
Grid Count: 0

